### ISSUE TITLE
Charcoal-filled Backpack transitions

### ISSUE THREAD

**Issue by Rukhmar** [jasonrohrer/OneLifeData7#1196]
As we have it right now, we can scoop up charcoal off the floor, and dump it on the floor in piles. But I was wondering if we could make it consistent with the uses we have for a Basket of Charcoal where we can add/remove charcoal from kilns and add into the distiller and various newcomen engines in the game as well. It just feels it would make sense since both the backpack and basket hold the same amount of charcoal, so it should be able to accomplish the same tasks. As we have it right now, we have to scoop it in the backpack, drop on the floor, then scoop it up with a basket to add it to any of the objects mentioned.

https://onetech.info/4893-Charcoal-filled-Backpack

https://onetech.info/298-Basket-of-Charcoal

**Comment by DiscardedSlinky** [comment]
Yeah that seems really annoying and should be an easy fix hopefully

**Comment by Hopie-OHOL** [comment]
Agreed, it seems like it just makes sense to allow backpacks to take charcoal in and out of kilns if they can pick charcoal up on the floor.

**Comment by Tarr-OHOL** [comment]
Might as well, it's like a 5 minute fix. Only needs 13 transitions to make basket of charcoal and backpack the same.

**Comment by Butterbraugh** [comment]
Yes! Please make this change. I was very excited when you added charcoal backpacks to the game. But then I was really disappointed to learn they couldn't be used on the kiln or forge. 
Sadly carrying charcoal in the backpack is basically pointless as it functions right now, because 99% of interaction with charcoal involves either adding it to a forge or Newcomen boiler or removing it from a kiln.
